ts/features/payments/history/types/index.ts

```diff
@@ -1,19 +1,33 @@
 import { NewTransactionResponse } from "../../../../../definitions/pagopa/ecommerce/NewTransactionResponse";
 import { PaymentRequestsGetResponse } from "../../../../../definitions/pagopa/ecommerce/PaymentRequestsGetResponse";
 import { RptId } from "../../../../../definitions/pagopa/ecommerce/RptId";
+import { WalletInfo } from "../../../../../definitions/pagopa/walletv3/WalletInfo";
+import { WalletInfo as EcommerceWalletInfo } from "../../../../../definitions/pagopa/ecommerce/WalletInfo";
 import { PaymentStartOrigin } from "../../checkout/types";
 import { WalletPaymentOutcomeEnum } from "../../checkout/types/PaymentOutcomeEnum";
 import { WalletPaymentFailure } from "../../checkout/types/WalletPaymentFailure";
+import { PaymentAnalyticsSelectedPspFlag } from "../../checkout/types/PaymentAnalyticsSelectedMethodFlag";
+import { Bundle } from "../../../../../definitions/pagopa/ecommerce/Bundle";
 
 export type PaymentHistory = {
   startOrigin?: PaymentStartOrigin;
   rptId?: RptId;
   startedAt?: Date;
   lookupId?: string;
+  savedPaymentMethods?: ReadonlyArray<WalletInfo | EcommerceWalletInfo>;
+  savedPaymentMethodsUnavailable?: ReadonlyArray<
+    WalletInfo | EcommerceWalletInfo
+  >;
+  pspList?: ReadonlyArray<Bundle>;
+  selectedPsp?: Bundle;
+  selectedPspFlag?: PaymentAnalyticsSelectedPspFlag;
+  selectedPaymentMethod?: string;
   verifiedData?: PaymentRequestsGetResponse;
+  formattedAmount?: string;
   transaction?: NewTransactionResponse;
   outcome?: WalletPaymentOutcomeEnum;
   failure?: WalletPaymentFailure;
   success?: true;
   attempt?: number;
+  serviceName?: string;
```

Comment:
This payload should not be updated, it is used for payment history persistence and Zendesk payload data. Could you consider using a dedicated type?

Reply:
I completely agree with you. I have created a new additional attribute with a different type in this commit as suggested by you: df78ebaaa48aa80f8c585f89c7883e85638bd1ad